.github/workflows/_run_super_scaffolding_tests.yml

```diff
@@ -48,23 +58,34 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+        with:
+          path: tmp/starter
+
+      - name: Checkout Core Repo
```

Comment:
I see that we have `Link Core Repo` in `_run_tests.yml`, should we have that here too? Is this file fine without it?

Reply:
Yep, we do need that here as well! Thanks for the review and keen eye, @gazayas.